### DISCUSSION TITLE
tests/exFat.img

### DISCUSSION THREAD

**Discussion by sajvn0x** [obhq/exfat#9] [General]
How you created the _tests/exFat.img_ file, What I have to learn to understand how that img file works. and what are things contained in the img file??

**Comment by ultimaweapon** [comment]
You can create it from Linux. Here are the steps I'm using:

- Use `dd` command to create a blank 1MB file from `/dev/zero`.
- Use `mkfs.exfat` command to format the file as exFAT.
- Use `mount` command with option `loop` to mount the file and put some file to it.
